101:
  connections:
    - id: 109
      cost: 5.0
    - id: 201
      cost: 1.0
    - id: 111
      cost: 5.0
    - id: 112
      cost: 4.0
  floor_id: 1
  x: 7.5219659805297852
  y: 0.31122153997421265
102:
  connections:
    - id: 101
      cost: 7.846331
    - id: 113
      cost: 4.0
    - id: 112
      cost: 4.0
  floor_id: 1
  x: 10.064212799072266
  y: 1.4873642921447754
103:
  connections:
    - id: 102
      cost: 36.066174
  floor_id: 1
  x: 9.089787483215332
  y: 7.4132962226867676
104:
  connections:
    - id: 103
      cost: 30.765376
  floor_id: 1
  x: 8.5161056518554688
  y: 12.930203437805176
105:
  connections:
    - id: 104
      cost: 20.505131
  floor_id: 1
  x: 4.0670309066772461
  y: 13.773331642150879
106:
  connections:
    - id: 105
      cost: 26.930959
  floor_id: 1
  x: 0.705474853515625
  y: 9.8197383880615234
107:
  connections:
    - id: 106
      cost: 16.630534
  floor_id: 1
  x: 0.69744110107421875
  y: 5.7416911125183105
108:
  connections:
    - id: 107
      cost: 9.333996
  floor_id: 1
  x: -0.081220626831054688
  y: 2.7874259948730469
109:
  connections:
    - id: 108
      cost: 12.259652
  floor_id: 1
  x: 1.8160848617553711
  y: -0.15534210205078125
110:
  connections:
    - id: 109
      cost: 19.833815
  floor_id: 1
  x: 1.553558349609375
  y: -4.6011147499084473
111:
  connections:
    - id: 110
      cost: 18.981140
  floor_id: 1
  x: 5.7249183654785156
  y: -3.343778133392334
112:
  connections:
    - id: 111
      cost: 17.580106
  floor_id: 1
  x: 9.6647577285766602
  y: -4.7782716751098633
113:
  connections:
    - id: 112
      cost: 76.823813
  floor_id: 1
  x: 13.130267143249512
  y: 3.2724461555480957
114:
  connections:
    - id: 104
      cost: 20.0 
  floor_id: 1
  x: 13.584233283996582
  y: 12.306552886962891
115:
  connections:
    - id: 113
      cost: 45.104379
    - id: 117
      cost: 79.036739
  floor_id: 1
  x: 14.87602710723877
  y: 0.33009064197540283
116:
  connections:
    - id: 118
      cost: 114.578611
  floor_id: 1
  x: 16.645235061645508
  y: -3.0534746646881104
117:
  connections:
    - id: 116
      cost: 10.843290
  floor_id: 1
  x: 16.261030197143555
  y: 0.21694988012313843
118:
  connections:
    - id: 117
      cost: 79.036739
  floor_id: 1
  x: 16.351036071777344
  y: -8.6728553771972656
119:
  connections:
    - id: 118
      cost: 30.258492
  floor_id: 1
  x: 17.346931457519531
  y: -14.082724571228027

201:
  connections:
    - id: 209
      cost: 5.0
    - id: 211
      cost: 5.0
    - id: 212
      cost: 4.0
  floor_id: 2
  x: 7.5219659805297852
  y: 0.31122153997421265
202:
  connections:
    - id: 201
      cost: 7.846331
    - id: 213
      cost: 4.0
    - id: 212
      cost: 4.0
  floor_id: 2
  x: 10.064212799072266
  y: 1.4873642921447754
203:
  connections:
    - id: 202
      cost: 36.066174
  floor_id: 2
  x: 9.089787483215332
  y: 7.4132962226867676
204:
  connections:
    - id: 203
      cost: 30.765376
  floor_id: 2
  x: 8.5161056518554688
  y: 12.930203437805176
205:
  connections:
    - id: 204
      cost: 20.505131
  floor_id: 2
  x: 4.0670309066772461
  y: 13.773331642150879
206:
  connections:
    - id: 205
      cost: 26.930959
  floor_id: 2
  x: 0.705474853515625
  y: 9.8197383880615234
207:
  connections:
    - id: 206
      cost: 16.630534
  floor_id: 2
  x: 0.69744110107421875
  y: 5.7416911125183105
208:
  connections:
    - id: 207
      cost: 9.333996
  floor_id: 2
  x: -0.081220626831054688
  y: 2.7874259948730469
209:
  connections:
    - id: 208
      cost: 12.259652
  floor_id: 2
  x: 1.8160848617553711
  y: -0.15534210205078125
210:
  connections:
    - id: 209
      cost: 19.833815
  floor_id: 2
  x: 1.553558349609375
  y: -4.6011147499084473
211:
  connections:
    - id: 210
      cost: 18.981140
  floor_id: 2
  x: 5.7249183654785156
  y: -3.343778133392334
212:
  connections:
    - id: 211
      cost: 17.580106
  floor_id: 2
  x: 9.6647577285766602
  y: -4.7782716751098633
213:
  connections:
    - id: 212
      cost: 76.823813
  floor_id: 2
  x: 13.130267143249512
  y: 3.2724461555480957
214:
  connections:
    - id: 204
      cost: 20.0 
  floor_id: 2
  x: 13.584233283996582
  y: 12.306552886962891
215:
  connections:
    - id: 213
      cost: 45.104379
    - id: 217
      cost: 79.036739
  floor_id: 2
  x: 14.87602710723877
  y: 0.33009064197540283
216:
  connections:
    - id: 218
      cost: 114.578611
  floor_id: 2
  x: 16.645235061645508
  y: -3.0534746646881104
217:
  connections:
    - id: 216
      cost: 10.843290
  floor_id: 2
  x: 16.261030197143555
  y: 0.21694988012313843
218:
  connections:
    - id: 217
      cost: 79.036739
  floor_id: 2
  x: 16.351036071777344
  y: -8.6728553771972656
219:
  connections:
    - id: 218
      cost: 30.258492
  floor_id: 2
  x: 17.346931457519531
  y: -14.082724571228027
